patches/tModLoader/Terraria/ModLoader/ModDust.cs

```diff
@@ -92,18 +93,14 @@ public class ModDust:ModTexturedType
 			}
 		}
 
-		public sealed override void Load()
-		{
-			base.Load();
-			Texture2D = !string.IsNullOrEmpty(Texture) ? ModContent.GetTexture(Texture).Value : TextureAssets.Dust.Value;
-		}
-
-		internal sealed override void AddInstance() {
+		public override void Load(Mod mod) {
+			base.Load(mod);
```

Comment:
Imagine not calling base.Load now on your overriding class...... oof (this is pretty vulnerable)

Reply:
Yeah, it may result in a null ref inside tML's code, which even intermediate modders won't be able to figure out or guess the reason of.